pkg/http/api.go

```diff
@@ -39,7 +39,7 @@ type api struct {
 	secretStores          map[string]secretstores.SecretStore
 	json                  jsoniter.API
 	actor                 actors.Actors
-	pubSub                pubsub.PubSub
+	publishFn             func(req *pubsub.PublishRequest) error
 	sendToOutputBindingFn func(name string, req *bindings.WriteRequest) error
```

Comment:
If it is ok to define the type for function and use it then, we can also create type for "func(name string, req *bindings.WriteRequest) error" as well ..

Reply:
That's a good idea, but I'd save it for a later refactor which would account generally for how we deal with function parameters globally.